components/filter/test/filter.visual-diff.js

```diff
@@ -228,7 +228,7 @@ describe('d2l-filter', () => {
 
 		describe('opener', () => {
 			after(async() => {
-				await page.evaluate(() => document.querySelector('html').setAttribute('lang', 'en'));
+				await page.reload({ waitUntil: ['networkidle0', 'load'] });
```

Comment:
Moving the resolution of `_noSelectionText` from `updated` to `willUpdate` seemingly caused the filter tests that followed `multiple-over-99` to show Arabic. I tried adding a delay but it didn't help. The `reload` did however. Really looking forward to having better test isolation.